packages/teams-js/src/public/dialog.ts

```diff
@@ -117,7 +117,9 @@ export namespace dialog {
   /**
    * Submit the dialog module and close the dialog
    *
-   * @param result - The result to be sent to the bot or the app. Typically a JSON object or a serialized version of it
+   * @param result - The result to be sent to the bot or the app. Typically a JSON object or a serialized version of it,
+   *  If this function is called from a dialog while {@link M365ContentAction} is set in the context object by the host, result will be ignored
```

Comment:
Is this how it supposed to work? or is it a bug and we are writing comment to let customers know the result will be ignored? I am also not sure what M365ContentAction is.

nit: the @link is not linking anything.
![image](https://user-images.githubusercontent.com/88564543/197655550-94f3d9c6-58bc-4e9a-bbea-6389bd32df9a.png)

Reply:
this is a limitation of this new feature, currently, it isn't expected that this will be a problem, but we want to make developers aware of this.  Also, how do @links work then?  I actually am not sure

Reply:
I see.

@link works just fine since you have imported the M365ContentAction. Sorry for the confusion. I was checking the wrong thing